sdk/elixir/.changes/unreleased/Fixed-20230816-235838.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixed
+body: 'fix: fix connect always timeout'
```

Comment:
duplicate of `fix`, since the kind already exist, you could just tip

```suggestion
body: 'fix connect always timeout'
```

Reply:
Fixes on https://github.com/dagger/dagger/pull/5646/commits/ee4abc5dabb020cc3e264a8cbbc583dddb088f06